### PR TITLE
Crud products

### PR DESCRIPTION
**Pull request by kristinadurant**

### CODE REVIEW

File: client/src/components/AddProduct.jsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+  //fetching the ingredients
+  useEffect(() => {
+    axios
+      .get('/api/ingredients')
+      .then(({ data }) => {
+        setIngredients(data);
+      })
+      .catch((error) => console.log(error));
+  }, [setIngredients]);
```
Comment (naaltunian) [inline client/src/components/AddProduct.jsx:54]: ingredients?

File: client/src/components/AddProduct.jsx [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+  const handleSubmission = async (e) => {
+    const form = e.target;
+    e.preventDefault();
```
Comment (naaltunian) [inline client/src/components/AddProduct.jsx:58]: move e.preventDefault() to first line of function

File: client/src/components/AddProduct.jsx [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+      form.reset();
+    } catch (error) {
+      console.log(error);
+      //   swal('Oops!', 'Something went wrong');
+    }
+  };
+
+  const handleUpdate = async (e) => {
+    const form = e.target;
+    e.preventDefault();
```
Comment (naaltunian) [inline client/src/components/AddProduct.jsx:72]: same as above comment

File: client/src/components/AddProduct.jsx (hunk 1 above)
Comment (kristinadurant) [reply client/src/components/AddProduct.jsx:54]: Hey, when I change it to ingredients, it throws me in an endless loop. It is working as setIngredients, I don't know if something else would work?

File: client/src/components/AddProduct.jsx (hunk 2 above)
Comment (kristinadurant) [reply client/src/components/AddProduct.jsx:58]: Hey Nick, when I change the order of it, it makes my ingredients and tags preselected. So it is not working like I want to to behave when i switch it. If I am doing it wrong, please help figure out what is wrong. But now as it is, without switching, it is doing what I want it to. Can we merge it as it is so we can prevent further merge conflicts, we are still working on these files.

File: client/src/components/AddProduct.jsx (hunk 3 above)
Comment (kristinadurant) [reply client/src/components/AddProduct.jsx:72]: same hapends

**Review by naaltunian — APPROVED** [review]

**Pull request merged**